scripts/deploy.ts

```diff
@@ -44,30 +42,6 @@ async function main() {
     console.log("Writing genesis.json");
     fs.writeFileSync("genesis.json", JSON.stringify(configs, null, 4));
     console.log("Successsfully deployed", configs);
-
-    if (argv.pubkeys) {
-        const pubkeys = argv.pubkeys.split(",");
-        await registerPublicKeys(contracts.blsAccountRegistry, pubkeys);
-    }
-}
-
-async function registerPublicKeys(
```

Comment:
Remove the public key registration from the deployment stage. Since we want to do it with deposits